python/oneflow/test/modules/test_module.py

```diff
@@ -256,6 +330,17 @@ def forward(self, x):
         m = ConvBNModule()
         delattr(m, "bn")
 
+    @flow.unittest.skip_unless_1n1d()
+    def test_all_hooks(test_case):
+        arg_dict = OrderedDict()
+        arg_dict["test_fun"] = [_test_hooks]
+        arg_dict["backward_register_fn"] = [
+            "register_backward_hook",
+            "register_full_backward_hook",
+        ]
+        for arg in GenArgList(arg_dict):
+            arg[0](test_case, *arg[1:])
```

Comment:
这种简单的情况就不需要 Gen 了吧（成八股文了😂），直接这样更香：
```suggestion
        for hook in ["register_backward_hook", "register_full_backward_hook"]:
            _test_hooks(test_case, hook)
```